YoutubeDownloader.Android/YoutubeDownloader.Android.csproj

```diff
@@ -0,0 +1,82 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+	<OutputType>Exe</OutputType>
+	<TargetFramework>net9.0-android</TargetFramework>
+	<SupportedOSPlatformVersion>21</SupportedOSPlatformVersion>
+	<Nullable>enable</Nullable>
+	<ApplicationId>com.tyrrrz.youtubedownloader</ApplicationId>
+	<ApplicationVersion>1</ApplicationVersion>
+	<ApplicationDisplayVersion>1.0.0.0</ApplicationDisplayVersion>
```

Comment:
If it's not how it works already by default, can this just be `<ApplicationDisplayVersion>$(Version)</ApplicationDisplayVersion>`?